latest/index.bs

```diff
@@ -175,7 +175,186 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data. OME-NGFF tables are a compatible subset of the [AnnData model](https://github.com/theislab/anndata). OME-NGFF tables implement the X (central dense matrix), the obs table (annotations on the rows of X) and the var table (annotations on the columns of X).
+
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images by image ID.
+│
+└── 123.zarr
+    └── table                 # The table group is a container which holds a table that is compatible with a subset of AnnData.
+        │                     # The table group MAY be in the root of the zarr file.
+        ├── .zgroup           # The table group MAY be in an image, labels, or points group.
```

Comment:
this is really cool to see, and it'd be really important to have this flexibility of having tables both in root and in image/labels group.

Reply:
Good point. I removed the references to points and changed it to say that the tables group may be in root or another group.